src/System.Private.ServiceModel/tools/test/BridgeCertificateInstaller/project.json

```diff
@@ -6,7 +6,9 @@
     "System.IO.FileSystem": "4.0.0",
     "System.IO": "4.0.10",
     "System.Net.Http": "4.1.0-rc3-24109-00",
-    "System.Net.Primitives": "4.0.10"
+    "System.Net.Primitives": "4.0.10",
+    "System.ServiceModel.Http": "4.1.0-rc3-24109-00",
+    "System.ServiceModel.Primitives": "4.1.0-rc3-24109-00"
```

Comment:
Will this work? I feel that we'll probably need to rev the package to a later package anyway to take this change. We can merge this now though.